geojson_pydantic/types.py

```diff
@@ -4,12 +4,11 @@
 
 from pydantic import conlist
 
-NumType = Union[float, int]
```

Comment:
What if we just change to `Numtype = float` ? It might avoid breaking change 🤷‍♂️

Reply:
I suppose it could be left there as a helper for someone if they import it elsewhere. Not sure there is much value in that though. As its just abstracting that its a float. I would probably say its best to not keep it since its no longer a `Union` but I also don't really work on libraries with external users much either and it might be different for others. 

Looking at what's public on GitHub I can't find anyone who is importing it, just defining it themselves in places. But can't really prove a lot with that either.

Reply:
Setting `NumType = float` is semantically a breaking change anyways because the union is being removed, I'd prefer to remove it.